code/src/components/DirectionButtons.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+
+// ----------------------------------------------------------------
+
+export const DirectionButtons = ({ direction, action }) => {
+  const dispatch = useDispatch();
+  return (
+    <DirectionButton
+      onClick={(event) => dispatch(action(event))}
+      direction={direction}
+    >
+      <Arrow role="img" aria-label={direction + `-arrow`} direction={direction}>
+        ➤
+      </Arrow>
+      <DirectionText direction={direction}>Go {direction}</DirectionText>
+    </DirectionButton>
```

Comment:
This was such a sophisticated implementation of a very cool addition.  The arrows oriented around the screen made the game MUCH easier to understand from a user-perspective.  Way to go!